### DISCUSSION TITLE
Dreame L10 Pro stopped responding, buttons are flashing

### DISCUSSION THREAD

**Discussion by dmirecki** [Hypfer/Valetudo#2026] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, for a few months

### Your questions

Hi,

My Dreame L10 Pro suddenly stopped working. Three buttons are flashing all the time, the robot is not available via Wi-Fi, I cannot start it. The robot does not respond to any button (even the small reset button does not work!). I can turn it off by pressing the power button for 10 seconds, but when I turn it on again, nothing changes - the buttons are still flashing.

The robot was not used for 2 days, and when I returned home, I noticed the buttons were flashing - so it is not caused by, for example, a software update or physical damage. The robot worked without any problems for several months.

Does anyone has an idea how to solve the problem?

**Comment by Hypfer** [comment, marked as answer]
Unfortunately, your robot is likely dead.

The L10 and Z10 suffer from bad solder joints of the RAM chips, leading to them losing contact with the PCB.
If you still have warranty, you should RMA the robot. If you don't you can either buy a new PCB, a new robot or - if you have the skills for that - reflow the RAM chips (might be dangerous though).


You can verify that this is actually the problem by connecting to the UART and checking if it endlessly prints something along the lines of `HELLO! SBOOT is starting!` without anything else

**Reply by dmirecki** [reply]
After connecting to the UART, I only see 2 lines:
```
[352]HELLO! SBOOT is starting!
[355]sboot commit : ee0e26c
```

This is not exactly the error you wrote about earlier, but does it also mean a RAM problem?

**Reply by Hypfer** [reply]
Yes that's what I was referring to

**Reply by dmirecki** [reply]
Thanks for help, I will try to RMA the robot
